tools/scripts/build.ps1

```diff
@@ -0,0 +1,15 @@
+choco install ant
```

Comment:
This would be the first time a "github-only" file showed up in this directory. I'd like to limit the 'github-only' files to as few locations as possible. This raises an interesting question of whether these sort of build files should eventually be committed to the HG openjdk repo, in which case it wouldn't matter.

Pending that more global discussion, perhaps there is a better place to put it? In a new appveyor dir maybe? Not sure (would like to think about this for more time than I have today).

Maybe one of the Gluon folks have some thoughts on this?

Reply:
It's a non-trivial question. In general, I am leaning towards separating code and build logic, since there might be multiple usages of the same source.
In this case however, the file is intended to be added for test purposes, hence I think it makes sense to do it here.
It is also not a github specific file, so I think it makes sense adding it to the hg repository as well.